app/Console/Commands/ParseGeomBoundariesPitches.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Helpers\GeometryHelper;
+use App\Models\V2\PolygonGeometry;
+use App\Models\V2\ProjectPitch;
+use App\Models\V2\Projects\ProjectPolygon;
+use App\Services\PythonService;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\App;
+use Illuminate\Support\Facades\DB;
+
+class ParseGeomBoundariesPitches extends Command
+{
+    protected $signature = 'parse:pitch-boundaries {framework_key}';
+
+    protected $description = 'Parse string geojsons of projects and project pitches by framework_key';
+
+    public function handle()
+    {
+        $frameworkKey = $this->argument('framework_key');
+
+        $projectPitches = $this->getProjectPitches($frameworkKey);
+        $ids = $projectPitches->pluck('id')->toArray();
+        $this->info(json_encode($ids));
+        $bar = $this->output->createProgressBar(count($projectPitches));
+        $bar->start();
+
+        foreach ($projectPitches as $pitch) {
+            if ($pitch->proj_boundary && $pitch->proj_boundary !== 'null') {
+                $this->processProjectPitch($pitch);
+            }
+            $bar->advance();
+        }
+
+        $bar->finish();
+        $this->info("\nGeometry boundaries parsing completed.");
+    }
+
+    private function getProjectPitches($frameworkKey)
+    {
+        return ProjectPitch::whereHas('fundingProgramme', function ($query) use ($frameworkKey) {
+            $query->where('framework_key', $frameworkKey);
+        })
+        ->whereNotNull('proj_boundary')
+        ->where('proj_boundary', '!=', 'null')
```

Comment:
It's so obnoxious that we have the literal string value "null" in this column, but I checked and there are 369 in prod 🤷🏻